server/db/create_db.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# TODO convert to node when things slow down
+psql -c 'DROP DATABASE IF EXISTS chronicle;' -U postgres
```

Comment:
:-( I, um, thought that role should always exist? I guess I can just not specify -U and assume that the `$USER` superuser role is available?